userspace/libpman/src/lifecycle.c

```diff
@@ -28,6 +29,82 @@ int pman_open_probe() {
 	return 0;
 }
 
+int pman_prepare_progs_before_loading() {
+	char msg[MAX_ERROR_MESSAGE_LEN];
+	/*
+	 * Probe required features for each bpf program, as requested
+	 */
+	errno = 0;
+	for(int ev = 0; ev < PPM_EVENT_MAX; ev++) {
+		event_prog_t *progs = event_prog_table[ev];
+		int idx, chosen_idx = -1;
+		for(idx = 0; idx < MAX_FEATURE_CHECKS && progs[idx].name != NULL; idx++) {
+			bool should_disable = chosen_idx != -1;
+			if(!should_disable) {
+				if(progs[idx].feat > 0 &&
+				   libbpf_probe_bpf_helper(BPF_PROG_TYPE_RAW_TRACEPOINT, progs[idx].feat, NULL) ==
```

Comment:
The actual programs we use in modern ebpf is `BPF_PROG_TYPE_TRACING` even if it shouldn't change too much

```suggestion
				   libbpf_probe_bpf_helper(BPF_PROG_TYPE_TRACING, progs[idx].feat, NULL) ==
```

Reply:
You are right about `usrsockaddr_in`, probably i mislooked that!